dd-java-agent/instrumentation/hibernate/core-4.0/src/test/groovy/QueryTest.groovy

```diff
@@ -177,6 +180,7 @@ class QueryTest extends AbstractHibernateTest {
           tags {
             "$Tags.COMPONENT.key" "java-hibernate"
             "$Tags.SPAN_KIND.key" Tags.SPAN_KIND_CLIENT
+            // "$Tags.DB_STATEMENT.key" "from Value"
```

Comment:
oversight or not working?

Reply:
Thought I took that out. But that test doesn't work, same as the `query.iterate` test above.

Reply:
Took it out. `Iterate` is rather expensive as it could end up making several calls to the database for each record. I believe this may get pretty chatty/expensive... (`List` on the other hand just gets all the info in one call. Think `scroll` is similar to list but kinda truncated... I haven't fully confirmed this)

Reply:
`iterate` is still tested separately (below the first test testing a list of functions)